style.css

```diff
@@ -0,0 +1,53 @@
+body {
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    background-color: lightgray;
+}
+
+h1 {
+    font-size: 55px;
+}
+
+#main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#img-div {
+    display: flex;
+    flex-direction: column;
+   
+   
+    background-color: white;
+}
+
+#img {
+    
+    min-width: auto;
+    min-height: auto;
+    max-width: auto;
+}
+
+#img-div img {
+    flex-shrink: 1;
+    min-width: auto;
+    min-height: auto;
+    max-width: 100%;
+}
+
+#img-caption {
+    flex-shrink: 1;
+    flex-basis: auto;
+    text-align:center;
+}
+
```

Comment:
Too many repetitions for the same group of values. Code could be shorter either using selectors together like 
`#img,
#img-div,
#img-caption {
values...
}`

or by using a class that contains the repetitive values. For example, we could create a d-flex class and use it in all of those elements we'd like to be a flex container.

Reply:
- [ ] I kindly suggest you to write comments politely and add an explanation here as well🤗. Please refer to above-mentioned comments for details.

Reply:
Thank you! Will try again :smile: